thirdeye-server/src/main/java/ai/startree/thirdeye/resources/MaintenanceResource.java

```diff
@@ -328,6 +331,27 @@ private EnumerationItemDTO getExistingOrCreate(final Long alertId, final Enumera
     return existingOrCreated;
   }
 
+  @POST
+  @Path("/anomaly/index-ignored")
+  @Timed
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation("Update the ignored index based on anomaly labels for historical anomalies")
+  public Response updateIgnoreLabelIndex(
+      @ApiParam(hidden = true) @Auth final ThirdEyePrincipal principal
+  ) {
+    // skip already updated ignored index
+    final DaoFilter filter = new DaoFilter().setPredicate(Predicate.NEQ("ignored", true));
+    anomalyManager.filter(filter).stream()
+        .peek(anomaly -> authorizationManager.ensureCanEdit(principal, anomaly, anomaly))
+        .filter(this::isIgnored)
+        .forEach(anomalyManager::update);
+    return Response.ok().build();
+  }
+  private boolean isIgnored(final AnomalyDTO anomaly) {
```

Comment:
I don't know whether you are using the intellij code style? it should add a line break between methods.

Reply:
I have imported the code style, let me check why its not getting enforced.